compiler/src/dotty/tools/dotc/core/Contexts.scala

```diff
@@ -167,6 +170,9 @@ object Contexts {
     /** The sbt callback implementation if we are run from sbt, null otherwise */
     def sbtCallback: AnalysisCallback = store(sbtCallbackLoc)
 
+    /** A map from absolute path to VirtualFile if we are run from sbt, null otherwise */
```

Comment:
Is this actually only if ran from sbt, or anything using zinc?

Reply:
anything with zinc - I just followed convention of the comment above

Reply:
updated to say zinc